projects/common/src/user/user-info.service.ts

```diff
@@ -0,0 +1,49 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { catchError, tap } from 'rxjs/operators';
+import { LoggerService } from '../logger/logger.service';
+import { UserTraits } from '../telemetry/telemetry';
+import { InMemoryStorage } from '../utilities/browser/storage/in-memory-storage';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserInfoService {
+  public BASE_URL: string = '/user-preferences';
```

Comment:
nit: Not something that needs urgent update, but URL is mostly the whole link, we should have something like `USER_PREFERENCE_ENDPOINT` or something similar here.